domains/transfer/Transfer.API/API/Program.cs

```diff
@@ -46,7 +46,6 @@
 
 builder.Services.AddValidatorsFromAssembly(typeof(Program).Assembly);
 
-builder.Services.AddEndpointsApiExplorer();
```

Comment:
Why this deletion?

Reply:
Leftover change, jeg glemte at revert. Har reverted igen nu, i dette commit:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/6887727c54f064949cef39556bca9ba41485aa3f